docs/v3-ea-components/transport-types/http-transport.md

```diff
@@ -1,3 +1,135 @@
 # HTTP Transport
 
-_Guide coming soon_
+HttpTransport is used to fetch data from a Provider using HTTP(S) protocol. In order to use HttpTransport you need to provide two methods `prepareRequests` and `parseResponse` that will be used to build requests and parse responses respectively. 
+
+Example usage of HttpTransport
+```typescript  
+const transport = new HttpTransport<EndpointTypes>({  
+  prepareRequests: (params, config) => {  
+    return params.map((param) => {  
+      const symbol = param.symbol.toLowerCase()  
+      const url = `/price/${symbol}`  
+  
+      return {  
+        params: param  
+		request: {  
+          baseURL: config.API_ENDPOINT,  
+          url,  
+        },  
+      }  
+    })  
+  },  
+  parseResponse: (params, res, config) => {  
+    return res.data.map((result) => {  
+      return {  
+        params: { symbol: result.symbol},  
+        response: {  
+          data: {  
+            result: result.price,  
+          },  
+          result: result.price,  
+        },  
+      }  
+    })  
+  },  
+})
+```
+### Building requests
+**prepareRequests**  method  takes the list of currently active input parameters in the subscription set and adapter config and should return one or multiple request configs (_ProviderRequestConfig_) depending if Data Provider supports request batching or not.
+
+#### Request config structure
+*ProviderRequestConfig* is a object that contains two keys `params` and `request` . `params` is the input parameters for a request and whatever  is provided will be available in the **parseResponse** method. `request` is the *axios* request configuration where request url, request method, query or body params, headers and etc can be specified.
+
+#### Send batch request to Data Provider
+The example above assumes that Data Provider doesn't support batching and each active input parameter will initiate request to Data Provider. Bellow is the example of batch version of prepareRequests
+```typescript  
```

Comment:
More spacing

```suggestion
#### Send batch request to Data Provider

The example above assumes that the Data Provider doesn't support batching, and each active input parameter will initiate a request to Data Provider. Below is an example of a batch version of prepareRequests.

```typescript  
```

Reply:
> The example above assumes

Which example is it referring to?

Reply:
the very first `Example usage of HttpTransport` .  maybe we can rename it like that ? 'the first example' ?